<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>521</width>
      <height>431</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="cesar">
     <attribute name="title">
      <string>Cifrado Cesar</string>
     </attribute>
     <widget class="QLineEdit" name="datoEC1">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>40</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="encriptarC">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>70</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Encriptar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="desencriptarC">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>220</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Desencriptar</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="datoDC2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>190</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Encriptaci贸n</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Desencriptaci贸n</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>100</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Palabra encriptada:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>260</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Palabra desencritada:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="claveEC1">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>40</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="palabraEncC">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>100</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="claveDC2">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>190</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="palabreDesC">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>260</y>
        <width>291</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Cadena</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>20</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Clave numerico</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>170</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Cadena</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>170</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Clave numerico</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="vigenere">
     <attribute name="title">
      <string>Cifrado Vigenere</string>
     </attribute>
     <widget class="QLineEdit" name="datoEV1">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>40</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="encriptarV">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>70</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Encriptar</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="datoDV2">
      <property name="geometry">
       <rect>
        <x>132</x>
        <y>190</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="desencriptarVV">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>220</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Desencriptar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Encriptaci贸n</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Desencriptaci贸n</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="claveEV1">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>40</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Cadena</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>20</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Clave</string>
      </property>
     </widget>
     <widget class="QLabel" name="palabraEncVV">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>100</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_30">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>100</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Palabra encriptada:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="claveDV2">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>190</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_31">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>170</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Cadena</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_32">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>170</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Clave</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_33">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>260</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Palabra desencritada:</string>
      </property>
     </widget>
     <widget class="QLabel" name="palabraDesV">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>260</y>
        <width>291</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Como Tas</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="cifrado">
     <attribute name="title">
      <string>Disco de Cifrado</string>
     </attribute>
     <widget class="QPushButton" name="botonCifradoDCE">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>80</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Encriptar</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="DatoCifradoDCE">
      <property name="geometry">
       <rect>
        <x>122</x>
        <y>80</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Encriptaci贸n </string>
      </property>
     </widget>
     <widget class="QLabel" name="datoDCE">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>130</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>grejtewshwtysrg</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_36">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>130</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Palabra encriptada:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Cadena</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>270</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Palabra desencritada:</string>
      </property>
     </widget>
     <widget class="QLabel" name="datoDCD">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>270</y>
        <width>281</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>aetjyrujertht</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="DatoCifradoDCD">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>210</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_34">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>190</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Cadena</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BatonCifradoDCD">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>210</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Desencriptar</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Maquina de Enigma</string>
     </attribute>
     <widget class="QScrollBar" name="scroll1">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>40</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QScrollBar" name="scroll2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>80</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QScrollBar" name="scroll3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>120</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="textscroll1">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>40</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>A</string>
      </property>
     </widget>
     <widget class="QLabel" name="textscroll2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>80</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>A</string>
      </property>
     </widget>
     <widget class="QLabel" name="textscroll3">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>120</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>A</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="datosME">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>170</y>
        <width>311</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>220</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Respuesta:</string>
      </property>
     </widget>
     <widget class="QLabel" name="respuesta">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>240</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="correr">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>170</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Correr</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>270</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Valores de inicio:</string>
      </property>
     </widget>
     <widget class="QLabel" name="resInicio">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>300</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>240</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
